Xamarin.Forms.Platform.UAP/ColorExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Windows.UI;
+using Windows.UI.Xaml.Media;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	internal static class ColorExtensions
+	{
+		public static Windows.UI.Color GetContrastingColor(this Windows.UI.Color color)
+		{
+			var nThreshold = 105;
+			int bgLuminance = Convert.ToInt32(color.R * 0.2 + color.G * 0.7 + color.B * 0.1);
```

Comment:
I don't get why this require a `Convert`

Reply:
No idea, this is just a renaming of a method which was already there.